ably.d.ts

```diff
@@ -604,6 +609,11 @@ export interface CorePlugins {
    * A plugin capable of decoding `vcdiff`-encoded messages. For more information on how to configure a channel to use delta encoding, see the [documentation for the `@ably-forks/vcdiff-decoder` package](https://github.com/ably-forks/vcdiff-decoder#usage).
    */
   vcdiff?: any;
+
+  /**
+   * A plugin which allows the client to be the target of push notifications.
+   */
+  push?: unknown;
```

Comment:
```suggestion
  Push?: unknown;
```

should be uppercase `Push` I think, in the implementation we access it `plugins?.Push`  ([like here](https://github.com/ably/ably-js/pull/1775/files#diff-fe43cce9d2f61250a5486a4e011d587aa9e2daaaaa4742c8dda3e635481a00e0R35) and other places)